examples/gke/gke.sh

```diff
@@ -1,12 +1,38 @@
 #!/bin/bash
 set -e
 
+#########
+# Start #
+#########
+
 display_usage() {
 	echo "End-to-end deployment script for scylla on GKE."
-	echo "usage: $0 [GCP user] [GCP project] [GCP zone]"
+	echo "usage: $0 [GCP user] [GCP project] [GCP zone] [cluster name (optional)]"
     echo "example: $0 yanniszark@arrikto.com gke-demo-226716 us-west1-b"
 }
 
+case $# in
```

Comment:
If this script has any future it would be better to add proper flags.
As for validation even now it would be better to check if all variables are set instead of that.

Reply:
Yeah that makes sense. I can do a proper arg parse thing.

Reply:
It will make the script non backwards compatible though but that is fine I think.